test/e2e/feature/feature.go

```diff
@@ -432,6 +432,10 @@ var (
 	// TODO: remove when SELinuxMount feature gate is enabled by default.
 	SELinuxMountReadWriteOncePodOnly = framework.WithFeature(framework.ValidFeatures.Add("SELinuxMountReadWriteOncePodOnly"))
 
+	// SeparateDiskTest (SIG-node, used for testing separate container runtime filesystem)
+	// The tests need separate disk settings on nodes and separate filesystems in storage.conf
+	SeparateDiskTest = framework.WithFeature(framework.ValidFeatures.Add("SeparateDiskTest"))
```

Comment:
nit: for consistency with the rest of features' names
```suggestion
	SeparateDisk = framework.WithFeature(framework.ValidFeatures.Add("SeparateDiskTest"))
```